scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -1090,21 +1156,20 @@ def config_part(self):
         cvg_cnt, cvg = get_cvg_cnt_and_cvg(self)
     dev_count = 1
     config_dict = read_config(MOTR_SYS_CFG)
+
     for i in range(int(cvg_cnt)):
         cvg_item = cvg[i]
         try:
-            metadata_devices = cvg_item["metadata_devices"]
+            metadata_device = cvg_item["devices"]["metadata"]
         except:
             raise MotrError(errno.EINVAL, "metadata devices not found\n")
-        check_type(metadata_devices, list, "metadata_devices")
-        self.logger.info(f"\nlvm metadata_devices: {metadata_devices}\n\n")
-
+        check_type(metadata_device, str, "metadata_devices")
+        self.logger.info(f"\nlvm metadata_device: {metadata_device}\n\n")
         # Currently only one metadata device in one cvg
-        for device in metadata_devices:
-            ret = create_parts(self, dev_count, device)
-            if ret != 0:
-                return ret
-            dev_count += 1
+        ret = create_parts(self, dev_count, metadata_device)
```

Comment:
create partiion is not requried for cortx-M0, lets remove it from mini provisioner

Reply:
Done ed4d648

Reply:
Added comments